packages/protocol/contracts/tokenvault/ERC20Vault.sol

```diff
@@ -372,6 +374,14 @@ contract ERC20Vault is
             ctokenDecimal: ctoken.decimals
         });
     }
+
+    /// @notice Sets the authorized status of an address, only the owner can
+    /// call this function.
```

Comment:
Do we have to write this in every sub-contract?

Reply:
Can rearrange these if i make BaseNFTVault inheriting from the AuthorizationBase. Will do later.